index.js

```diff
@@ -107,24 +141,33 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
-//
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+console.log(myMap([1,2,3,4,5], triple)) 
+
+function triple(number){
+  console.log(number * 3)
```

Comment:
This also should be a `return` instead of a `console.log` - the purpose of the `myMap` function is to create a new array with each element based on the callback function, so that callback needs to return a value to be added to the new array. With this code, we'd get back an array of `[undefined, undefined, undefined...]` instead of `[3,6,9...]`